src/Analyzers/CSharp/CodeFixes/NewLines/EmbeddedStatementPlacement/EmbeddedStatementPlacementCodeFixProvider.cs

```diff
@@ -48,7 +48,7 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
         public static async Task<Document> FixAllAsync(Document document, ImmutableArray<Diagnostic> diagnostics, CodeActionOptionsProvider codeActionOptionsProvider, CancellationToken cancellationToken)
         {
             var root = await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
-            var editor = new SyntaxEditor(root, document.Project.Solution.Workspace.Services);
+            var editor = document.GetSyntaxEditor(root);
```

Comment:
these are at a layer (code_style) that cannot access internal apis.  So while the API is in flux and hasn't been fully signed off on, this code has to go through an indirection (an extension method) so it can do the right thing in Roslyn vs CodeStyle.